### ISSUE TITLE
Creating a template of specific workload object

### ISSUE THREAD

**Issue by Wopienkaatwork** [microsoft/Microsoft365DSC#4378]
Hi, 
is it possible to create a powershell file (a template) where you can see all the possible properties that can be set on an object? 
Like you have in the wiki docs but in a powershell form to give to somebody to fill out the desired parameters and I will put it into my devops solution. 

Is it possible to use the resource generator for this task?

**Comment by subhashvinjamuri** [comment]
Yes, im also looking for same. For now, creating it manually. A tedious work.

**Comment by Wopienkaatwork** [comment]
I found a first step: 
get-DscResource -Syntax TeamsCallingPolicy -Verbose which is just a string but it is a start

**Comment by andikrueger** [comment]
Have you had a look at the code of the Doc Generator: https://github.com/microsoft/Microsoft365DSC/blob/Dev/Modules/Microsoft365DSC/Modules/M365DSCDocGenerator.psm1 Maybe there is a chance to repurpose it a bit for your task.
